spdx-utils/build.gradle

```diff
@@ -51,7 +51,7 @@ def getLicenseHeader(fromYear = 2017, toYear = java.time.Year.now().value) {
      """.stripIndent()
 }
 
-def licenseToEnumEntry(String license) {
+def licenseToEnumEntry(license) {
```

Comment:
Or add types to the other functions :)

Reply:
That's actually what I did first, for type safety, but that made the signature of the new `downloadLicenseTexts` function very long... so I decided to go the other way, until we migrate to Kotlin build scripts 😉

Reply:
But that's cheating ;) If the function takes many arguments, so be it, I'd rather have a long signature than struggling to understand what the arguments are supposed to be.